.github/workflows/pull-transifex.yml

```diff
@@ -0,0 +1,28 @@
+name: "Pull translations from Transifex"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 3 * * *"
+
+jobs:
+  pull-translations:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ssh-key: ${{ secrets.DEPLOYMENT_SSH_KEY }}
+          ref: "master"
+
+      - name: l10n-pull
+        uses: transifex/cli-action@v2
+        with:
+          token: ${{ secrets.TX_TOKEN }}
+          args: pull --force --skip --all
```

Comment:
maybe display a diff? before the commit

Reply:
since everything is one big file this diff can be rather huge .... no idea if this is of any help ... :shrug: